diagrams/sequence-diagrams/creating-and-publishing-a-form.md

```diff
@@ -0,0 +1,28 @@
+# Creating and publishing a form
+
+```mermaid 
+sequenceDiagram
+  actor user as User
+  participant manager as Manager
+  participant designer as Designer
+  participant api as API
+
+  user -->> manager: Creates a new form
```

Comment:
We may want to reference an existing form (e.g. page on GOV.UK) that this is the online version of.